pygeohash/stats.py

```diff
@@ -9,14 +9,15 @@
 """
 
 import math
-from pygeohash.geohash import decode, encode
-from pygeohash.distances import geohash_haversine_distance
+from typing import Iterable
 
+from pygeohash.distances import geohash_haversine_distance
+from pygeohash.geohash import decode, encode
 
 __author__ = 'Will McGinnis'
 
 
-def mean(geohashes):
+def mean(geohashes: Iterable[str]):
```

Comment:
@wdm0006 : Considering that type hints are nearly completely ubiquitous in IDEs, would it be worth it to modify the docstrings to not have a redundant mention "iterable of geohashes"? I figured this way was fine still